### ISSUE TITLE
Change name of a tool when a brush is bound to it.

### ISSUE THREAD

**Issue by NJDaeger** [EngineHub/WorldEdit#1589]
**Is your feature request related to a problem? Please describe.**
I am a heavy user of brushes and can often be using all 9 slots of my inventory for different brushes. I often run into the issue of forgetting which brush does what- especially after I go afk for a little while and come back.

**Describe the solution you'd like**
I think it would be nice to modify the brush command to accept a greedy string at the end. The greedy string would specify a name for the brush and then set the name of the item you bound the brush to the name specified. The command syntax would follow what I have below.

`/brush <brushtype> <brushtype_specific_options> [name_to_set]`

So, if I were to create several clipboard brushes (each with a different object), I could have each of the brushes named accordingly. 

**Describe alternatives you've considered**
An alternative I have considered is simply just renaming the item to the type of brush you're using. However, I think this would not be as intuitive, because if I were using multiple brushes of the same type, they would all be named the same. 

**Additional context**
N/A

**Comment by wizjany** [comment]
this won't work in the current system since brushes aren't bound to a particular item stack, they are loosely bound to the item type for the player.
this probably will go along with #1306 as it may allow brushes to be bound to specific item stacks (via nbt identifiers or so) rather than to an item type. not sure what the plans are for that though.

that is to say, in the current system you can just use an anvil (or even vanilla's /item command) to just rename the items yourself, as worldedit doesn't care about the name at all.

**Comment by NJDaeger** [comment]
Yeah, they could just use the itemname command from essentials for now, I just think having it all be in a single command would be a bit nicer imo.

**Comment by wizjany** [comment]
i'm against adding the name there in WE until we have a full-fledged way to bind to nbt instead of to item type. it's terrible for UX to provide that as the user will think it's relevant to the brush being bound even though it's completely unrelated to the brush.

**Comment by NJDaeger** [comment]
Not adding it right this instant is fine- I'm just saying in general when the brush system gets to the point where they are bound to nbt instead, having it within the command would be nice.
